lib/parser/current.rb

```diff
@@ -84,7 +84,7 @@ def warn_syntax_deviation(feature, version)
     CurrentRuby = Ruby27
 
   when /^3\.0\./
```

Comment:
It looks like the `else` branch also needs to be updated.

```diff
  when /^3\.0\./
-   current_version = '3.0.0-dev'
+   current_version = '3.0.0'
    if RUBY_VERSION != current_version
      warn_syntax_deviation 'parser/ruby30', current_version
    end

    require 'parser/ruby30'
    CurrentRuby = Ruby30

  else # :nocov:
    # Keep this in sync with released Ruby.
-   warn_syntax_deviation 'parser/ruby27', '2.7.x'
-   require 'parser/ruby27'
-   CurrentRuby = Ruby27
+   warn_syntax_deviation 'parser/ruby30', '3.0.x'
+   require 'parser/ruby30'
+   CurrentRuby = Ruby30
  end
```

cf: https://github.com/whitequark/parser/commit/47111d191f6e989db562d2a08e92fa9da87c4435